src/safeposix/syscalls/fs_calls.rs

```diff
@@ -100,7 +100,11 @@ use crate::safeposix::cage::{FileDescriptor::*, *};
 use crate::safeposix::filesystem::*;
 use crate::safeposix::net::NET_METADATA;
 use crate::safeposix::shm::*;
-
+use crate::interface::log_to_stdout;
```

Comment:
You need to not import everything individually. We already import the interface, just call the individual functions using interface::log_to_stdout etc